### ISSUE TITLE
Timout On Activation

### ISSUE THREAD

**Issue by StephenHodgson** [mob-sakai/unity-activate#13]
**Describe the bug**
Happens occasionally.
```
Error: waiting for selector `#new_conversations_create_session_form #conversations_create_session_form_password` failed: timeout 2000ms exceeded
```

**Comment by StephenHodgson** [comment]
fixed by https://github.com/mob-sakai/unity-activate/pull/12

**Comment by mob-sakai** [comment]
🎉 This issue has been resolved in version 1.4.0 🎉
